blocks/identity-block/components/HeadlinedSubmitForm/styles.scss

```diff
@@ -8,7 +8,7 @@
 
 .xpmedia-form-submittable {
   border-bottom: 1px solid $border-rule-color;
-  margin-bottom: map-get($spacers, 'sm');
+  margin-bottom: map-get($spacers, 'md');
```

Comment:
were these changes to margin deliberate? I think @nschubach worked on that component? 

```suggestion
  margin-bottom: map-get($spacers, 'sm');
```

Reply:
Yep, at least the zeplin mocks I was looking at had different spacing
https://app.zeplin.io/project/603fa53e2626ed1592e7c0e6/screen/6113e22a4eb86998b18dbed8

Reply:
Bottom margin should be `md` (the space under the border).  I don't know why this was `sm`.
![image](https://user-images.githubusercontent.com/2287238/133159291-c0db7343-e76a-4a81-825c-4493050d0c58.png)
So this is correct given the third party buttons are below that.